calico_cni/policy_drivers.py

```diff
@@ -131,13 +131,55 @@ def generate_tags(self):
         """
         return []
 
+class MesosPolicyDriver(DefaultPolicyDriver):
+    """
+    Implements default network policy for a Mesos container manager.
+    """
+    def __init__(self, network_name, network_args):
+        """
+        Extract any labels that have been specified in the mesos-namespaced labels field
+        of the network_args. Mesos CNI inserts the following additional data into
+        the args field:
+        "args" : {
+          "org.apache.mesos" : {
+            "network_info" : {
+              "name" : "mynet",
```

Comment:
For my own understanding, the value in "network_name" will be the same as in network_info["name"], correct?